cmd/update/update.go

```diff
@@ -70,6 +73,16 @@ var UpdateCmd = &cobra.Command{
 			Runtime:     runtimeVersions,
 		}
 
+		if updateCmdOptions.check {
+			if !update.ComparePackages(hconf.Packages.Development, devVersions) || !update.ComparePackages(hconf.Packages.Runtime, runtimeVersions) {
+				fmt.Println(styles.WarnStyle.Render("Updates are available"))
+				os.Exit(1)
+			} else {
```

Comment:
can we drop the else statement here?
It'll have the same meaning as we're exiting

Reply:
should the print & the exit code stay?